src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
+
+  const count = days * price;
```

Comment:
`Count` should be renamed too. Let's say
```suggestion
  const fullPrice = days * price;
```